Makefile

```diff
@@ -41,6 +41,6 @@ remediation_test:
 upgrade_test:
 	make -C ./scripts/feature_tests/upgrade/
 
-feature_tests: setup_env remediation_test cleanup_env pivoting_test
+feature_tests: setup_env remediation_test cleanup_env upgrade_test cleanup_env pivoting_test
```

Comment:
there's a new target for upgrade so not included in 'feature_tests'
implemented in https://github.com/metal3-io/metal3-dev-env/pull/390